packages/roosterjs-editor-dom/lib/index.ts

```diff
@@ -49,6 +49,14 @@ export { default as readFile } from './utils/readFile';
 export { default as getInnerHTML } from './utils/getInnerHTML';
 export { default as setColor } from './utils/setColor';
 export { default as matchesSelector } from './utils/matchesSelector';
+export {
+    adjustInsertPositionForHyperLink,
+    adjustInsertPositionForStructuredNode,
+    adjustInsertPositionForParagraph,
+    adjustInsertPositionForVoidElement,
+    adjustInsertPositionForMoveCursorOutOfALink,

```

Comment:
I think we don't need to export these sub functions

Reply:
@JiuqingSong Won't they be used separed?

Reply:
At least for now we don't have such requirement. When we have we can then export them.
It is always easier to adding an export them removing since latter is a breaking change.